### ISSUE TITLE
App Crashes on Accessing "Hamburger" Menu

### ISSUE THREAD

**Issue by TrentSaggus** [nextcloud/android#9020]
### Steps to reproduce
1. Install Next Cloud (or dev stream, doesn't matter) from F-Droid on a CalyxOS
2. Open app and use then close, "Hamburger" menu works just fine.
3. Close App
4. Reopen app and attempt to access "Hamburger" menu. Repeat as many times as you wish, outcome is the same.

### Expected behaviour
- App should not crash at step 4. Menu should be displayed and accessible.

### Actual behaviour
- App crashes at step 4 and provides log as the below.

### Can you reproduce this problem on https://try.nextcloud.com?
- Not a server issue. Only occurs on any device using CalyxOS
EDIT:
- Other android device (OnePlus 7 Pro) has zero Issues when connecting to same server.

### Environment data
Android version: 
11 (CalyxOS 2.9.0)

Device model: 
Pixel 3a

Stock or customized system:
Stock

Nextcloud app version:
30170090

Nextcloud server version:
21.0.1.1

Reverse proxy:
Nil

#### Nextcloud log (data/nextcloud.log)  #### This is from the app as it crashes!
```
\*\*\*\*\*\*\*\*\*\* **CAUSE OF ERROR** \*\*\*\*\*\*\*\*\*\*

java.lang.RuntimeException: Canvas: trying to draw too large(150896656bytes) bitmap.

	at android.graphics.RecordingCanvas.throwIfCannotDraw(RecordingCanvas.java:280)

	at android.graphics.BaseRecordingCanvas.drawBitmap(BaseRecordingCanvas.java:88)

	at android.graphics.drawable.BitmapDrawable.draw(BitmapDrawable.java:548)

	at android.graphics.drawable.LayerDrawable.draw(LayerDrawable.java:1018)

	at android.widget.ImageView.onDraw(ImageView.java:1436)

	at android.view.View.draw(View.java:22350)

	at android.view.View.updateDisplayListIfDirty(View.java:21226)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.draw(View.java:22353)

	at android.view.View.updateDisplayListIfDirty(View.java:21226)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.updateDisplayListIfDirty(View.java:21217)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at androidx.recyclerview.widget.RecyclerView.drawChild(RecyclerView.java:5030)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.draw(View.java:22353)

	at androidx.recyclerview.widget.RecyclerView.draw(RecyclerView.java:4429)

	at android.view.View.updateDisplayListIfDirty(View.java:21226)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.draw(View.java:22353)

	at com.google.android.material.internal.ScrimInsetsFrameLayout.draw(ScrimInsetsFrameLayout.java:109)

	at android.view.View.updateDisplayListIfDirty(View.java:21226)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at androidx.drawerlayout.widget.DrawerLayout.drawChild(DrawerLayout.java:1426)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.draw(View.java:22353)

	at android.view.View.updateDisplayListIfDirty(View.java:21226)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.updateDisplayListIfDirty(View.java:21217)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.updateDisplayListIfDirty(View.java:21217)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.updateDisplayListIfDirty(View.java:21217)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.updateDisplayListIfDirty(View.java:21217)

	at android.view.View.draw(View.java:22081)

	at android.view.ViewGroup.drawChild(ViewGroup.java:4516)

	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)

	at android.view.View.draw(View.java:22353)

	at com.android.internal.policy.DecorView.draw(DecorView.java:806)

	at android.view.View.updateDisplayListIfDirty(View.java:21226)

	at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:559)

	at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:565)

	at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:642)

	at android.view.ViewRootImpl.draw(ViewRootImpl.java:4106)

	at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:3833)

	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3104)

	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1948)

	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8177)

	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:972)

	at android.view.Choreographer.doCallbacks(Choreographer.java:796)

	at android.view.Choreographer.doFrame(Choreographer.java:731)

	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)

	at android.os.Handler.handleCallback(Handler.java:938)

	at android.os.Handler.dispatchMessage(Handler.java:99)

	at android.os.Looper.loop(Looper.java:223)

	at android.app.ActivityThread.main(ActivityThread.java:7664)

	at java.lang.reflect.Method.invoke(Native Method)

	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)

	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

\*\*\*\*\*\*\*\*\*\* **APP INFORMATION** \*\*\*\*\*\*\*\*\*\*

ID: com.nextcloud.client

Version: 30170090

Build flavor: generic

\*\*\*\*\*\*\*\*\*\* **DEVICE INFORMATION** \*\*\*\*\*\*\*\*\*\*

Brand: Android

Device: bonito

Model: Pixel 3a XL

Id: RQ3A.210905.001

Product: calyx_bonito

\*\*\*\*\*\*\*\*\*\* **FIRMWARE** \*\*\*\*\*\*\*\*\*\*

SDK: 30

Release: 11

Incremental: 202109290
```

**Comment by TrentSaggus** [comment]
Sorry for all the edits, I keep thinking of more information to provide.

Troubleshooting steps taken:
- I have also attempted to uninstall and re-install the app from F-Droid. Same results.
- I have reinstalled base OS and reinstalled app. Same results.
- I have installed from Aurora store through Play Store. Same results.
- Attempted to replicate issue on OnePlus device without success.

If someone could let me know a work around until a permanent fix is found that would be amazing.

Thank you so much for a great platform!
